src/Scripting/Core/ScriptingResources.resx

```diff
@@ -1,103 +1,122 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-<!-- 
-	Microsoft ResX Schema
+  <!-- 
+    Microsoft ResX Schema 
+    
+    Version 2.0
```

Comment:
I asume its because the file wasn't changed since 29 Dec 15. Now with the next VS version a new format has been used automatically since I added a new record.